spec/factories/contacts.rb

```diff
@@ -5,5 +5,12 @@
     email
     c.public { [true, false].sample }
     text { generate(:description) }
+
+    trait :with_message do
+      send_name "David Wessman"
+      send_email "spindelman@fsektionen.se"
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.